core/src/labels/spriteLabel.cpp

```diff
@@ -1,15 +1,35 @@
 #include "labels/spriteLabel.h"
+#include "platform.h"
 
 namespace Tangram {
 
-SpriteLabel::SpriteLabel(Label::Transform _transform, glm::vec2 _size,
-                         LabelMesh& _mesh, int _vertexOffset, Label::Options _options) :
-    Label(_transform, _size, Label::Type::point, _mesh, {_vertexOffset, 4}, _options) {}
+SpriteLabel::SpriteLabel(Label::Transform _transform, glm::vec2 _size, LabelMesh& _mesh,
+                         int _vertexOffset, Label::Options _options, glm::vec2 _anchor) :
+    Label(_transform, _size, Label::Type::point, _mesh, {_vertexOffset, 4}, _options)
+ {
+    m_anchor = _anchor;
```

Comment:
If you call the constructor of the base class initializer lists don't apply (c++11), except if this is a c++14 feature.